header_struct_types.m

```diff
@@ -0,0 +1,13 @@
+// Header Struct Types
+// This will create the struct for the Interface File
+//
+// IDECodeSnippetCompletionPrefix: vokStructInterface
+// IDECodeSnippetCompletionScopes: [TopLevel]
+// IDECodeSnippetIdentifier: 92CFC1EC-C7B9-4960-834F-31C0878FFE84
+// IDECodeSnippetLanguage: Xcode.SourceCodeLanguage.Objective-C
+// IDECodeSnippetUserSnippet: 1
+// IDECodeSnippetVersion: 2
+FOUNDATION_EXPORT const struct <#XXXStructName#> {
+    __unsafe_unretained <#Type#> *<#variableName1#>;
+    __unsafe_unretained <#Type#> *<#variableName2#>;
```

Comment:
I just figured most times you want a struct your going to have at least two items in it.  But since it doesn't take much effort to copy and paste for more items I can change to have a more streamlined approach if that is preferable.